examples-next/images/types.ts

```diff
@@ -0,0 +1,38 @@
+type ImageMode = 'local' | 'cloud';
```

Comment:
re: above, I would change this type to `Storage: 'filesystem' | 'cloud'` and would change the `mode` key in all following types to `storage`